### PR TITLE
Release version 5.0.0

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Release version 5.0.0
| Type?             | 
| BC breaks?        | 
| Deprecations?     | 
| Fixed ticket?     | https://github.com/PrestaShop/PrestaShop/issues/31931
| Sponsor company   | 
| How to test?      |

### CODE REVIEW

**Review by matthieu-rolland — COMMENTED** [review]
so what are the conditions for this release PR to be pushed forward ? 

There should be testing by QA before release as well

**Comment by Hlavtox** [comment]
@matthieu-rolland Approve and ready to go. :-)

We agreed with @florine2623 that we will do full test on the release PR. 👍

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ Gain clear insights into important metrics about your customers, using Google An
 
 To use it, you will need to create a Google Analytics account and insert your Google Analytics Identifier into the Module configuration page.
 
+## Compatibility
+
+PrestaShop: `1.7.6.0` or later
+
```
Comment (leemyongpakvn) [inline README.md:12]: I forgot to change min compat. here, maybe @florine2623 can help us ;)

**Review by ShaiMagal — APPROVED** [review]

**Review by florine2623 — COMMENTED** [review]
Hello @Hlavtox ,

Tested on `develop`.

Tested https://github.com/PrestaShop/ps_googleanalytics/pull/153.
I used Google Assistant to check if all the items for Hits sent are displayed.

Screenshot 2024-01-10 at 15 14 14

⚠️ There are some that are missing.
- I can't trigger the `sign_up` event (I've created a new customer account) :x:
- the `cart adding/increasing/decreasing/removing` :x:
Could you check ? Or maybe I am missing something ? 😅 
- All the other items are well present. ✅ 

Tested https://github.com/PrestaShop/ps_googleanalytics/pull/150.


 - Payment info now using order currency not context ✅ 
Screenshot 2024-01-10 at 15 19 12

- Added proper loading of manual orders created in backoffice :x: 
I can't see the order created from BO in the Google Assistant. The Tag Assistant is well linked to the BO.
Screenshot 2024-01-10 at 15 25 40

I can't find any info on the order I created from the BO. Only this is displayed from the BO order creation action :
Screenshot 2024-01-10 at 15 26 55

Tested the module google analytics in general. LGTM.
Screenshot 2024-01-10 at 15 30 41

Waiting for your feedback ^^
Thanks!

**Comment by ChronoBrake** [comment]
any news when it will be released?

**Comment by Hlavtox** [comment]
@florine2623 Tested everything now.
 
> I can't trigger the sign_up event (I've created a new customer account) ❌

This is triggered by vising registration page in FO. We can improve it later so it also triggers when creating a new account in checkout. :-)

> the cart adding/increasing/decreasing/removing ❌ Could you check ? Or maybe I am missing something ? 😅

These are always sent on the next page load. You add/remove things in cart, then on the next proper load, all these events are sent.

> Added proper loading of manual orders created in backoffice ❌
> I can't see the order created from BO in the Google Assistant. The Tag Assistant is well linked to the BO.

I think it must have been some random issue or you just missed it in the events, tried it and works ;-)
Snímek obrazovky 2024-01-29 212730

**Comment by fatihdursun** [comment]
Hi @Hlavtox When will the new version be released?

**Comment by Hlavtox** [comment]
@florine2623 One more thing - the add/remove from cart events may have not been firing because of a core issue, fixed here - https://github.com/PrestaShop/PrestaShop/pull/35215.

**Comment by ChronoBrake** [comment]
If it will help, we tested that also from dev build and all working fine on prestashop 8.1.3

**Comment by kpodemski** [comment]
@ChronoBrake 

Please be patient, as far as I know you are not paying us to work for you.

**Comment by Hlavtox** [comment]
Bump @PrestaShop/qa-functional

**Review by florine2623 — APPROVED** [review]
Hello all,

Sorry for the delay ^^

I reviewed your comment @Hlavtox , noted for the first 2. And for the last one, I checked again, ineed it works as expected. I think last time I forgot to enable this option in the BO :
Screenshot 2024-02-08 at 13 51 06

It works as expected, the new order page from the BO is counted in Google analytics :

Screenshot 2024-02-08 at 13 57 57

**Comment by Hlavtox** [comment]
No worries, thank you @florine2623!!!!!!! ❤️

**Pull request merged**